preview/action.yaml

```
@@ -24,6 +24,10 @@ inputs:
     description: "Read the Docs Community or Read the Docs for Business (community or business)"
     default: "community"
     required: false
+  single_version:
+    description: "Site is a single verison project, so link to the top level."
+    default: 'false' 
```

is there a specific reason why you are using a string here instead of a boolean?

```suggestion
    default: false
```

---

It's documented as a string in the docs, so I assumed it wouldn't be supported. If there's a good way to test this before merging it, I'd love to know how, and we should probably add that to the docs.

---

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault

---

🙃 https://github.com/actions/runner/issues/1483

---

Also here, they use `type: boolean` but then they check for `'true'` 😄 https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/